openapi/definitions.yaml

```diff
@@ -795,6 +806,7 @@ ServiceMetadata:
   allOf:
     - $ref: "#/StandardServiceMetadata"
     - $ref: "#/SpecialServiceMetadata"
+    - $ref: "#/CommonServiceMetadata"
```

Comment:
A little bit tricky, but i can not think a better way with openapi 2.x. 
Maybe we could simplify this polimorfism when we'll upgrade to 3.x.